XamarinCommunityToolkit/Views/SegmentedView/iOS/SegmentedViewRenderer.ios.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using Xamarin.CommunityToolkit.iOS.UI.Views;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using UIKit;
+using Xamarin.Forms.Platform.iOS;
+using System.Threading.Tasks;
+using System.Threading;
+using Foundation;
+using Xamarin.Forms.Internals;
+using Xamarin.CommunityToolkit.Extensions.iOS;
+
+[assembly: ExportRenderer(typeof(SegmentedView), typeof(SegmentedViewRenderer))]
+
+namespace Xamarin.CommunityToolkit.iOS.UI.Views
+{
+	[Xamarin.Forms.Internals.Preserve(AllMembers = true)]
+	public class SegmentedViewRenderer : ViewRenderer<SegmentedView, UISegmentedControl>
+	{
+		protected override void OnElementChanged(ElementChangedEventArgs<SegmentedView> e)
+		{
+			base.OnElementChanged(e);
+
+			if (e.NewElement == null)
+				return;
+
+			if (Control == null)
+				SetNativeControl(new UISegmentedControl());
+
+			if (e.OldElement != null && Control != null)
+				InvalidateControl();
+
+			if (e.NewElement != null)
+				InitializeControl();
+		}
+
+		void InitializeControl()
+		{
+			PopulateSegments(Element.Items);
+			Control.ClipsToBounds = true;
+			Control.SelectedSegment = Element.SelectedIndex;
+			Control.BackgroundColor = Element.BackgroundColor.ToUIColor();
+			Control.Layer.MasksToBounds = true;
+			UpdateSelectedSegment(Element.SelectedIndex);
+
+			Control.ValueChanged += OnSelectedIndexChanged;
+			((INotifyCollectionChanged)Element.Items).CollectionChanged += SegmentsCollectionChanged;
+
+			if (Element.IsColorSet)
+			{
+				Control.SelectedSegmentTintColor = Element.Color.ToUIColor();
+				Control.TintColor = Element.Color.ToUIColor();
+			}
+		}
+
+		void PopulateSegments(IList<string> segments)
+		{
+			for (var i = 0; i < segments.Count; i++)
+			{
+				InsertSegment(segments.ElementAt(i), i);
+			}
+		}
+
+		async void InsertSegment(string segment, int position)
+		{
+			switch (Element.DisplayMode)
+			{
+				case SegmentMode.Image:
+					var img = await ((ImageSource)segment).GetNativeImageAsync();
+					if (img != null)
+						Control.InsertSegment(img, position, false);
+					else
+						Console.WriteLine("ImageSource is null");
```

Comment:
I think that we use a Log here, to make it more visible to devs. Can you use the `Log` as you did in line 163?